lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -47,6 +47,7 @@ abstract class AssetPickerBuilderDelegate<Asset, Path> {
     this.pathNameBuilder,
     this.assetsChangeCallback,
     this.assetsChangeRefreshPredicate,
+    this.enableDragAndSelect = true,
```

Comment:
We might only want the feature with the default implementation.

Reply:
Ah what I mean is the feature should be placed in the `DefaultAssetPickerBuilderDelegate`.